courses/free-courses-hi.md

```diff
@@ -344,6 +344,7 @@
 * [Machine Learning Full Course](https://www.youtube.com/watch?v=IoZGSQ07e8g) - Bharani Akella, Great Learning (YouTube)
 * [Machine Learning in Hindi](https://www.youtube.com/playlist?list=PLPbgcxheSpE0aBsefANDYe2X_-tyJbBMr) - Codebasics Hindi
 * [Machine Learning Tutorials For Beginners Using Python in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ai6fAMHp-acBmJONT7Y4BSG) - CodeWithHarry (YouTube)
+* [Machine Learning Project Videos with Resources] (https://t.me/joinmlprojects) - Telegram Channel
```

Comment:
This is not a freely available resource and there is no information about Machine Learning in the specified link.